scripts/build/docker/compile-template.js

```diff
@@ -31,7 +31,7 @@ const template = fs.readFileSync(path.join(currentDirectory, 'Dockerfile.templat
 _.each([
   {
     architecture: 'i686',
-    image: 'erwinchang/ubuntu-14.04-32bit-build'
+    image: 'erwinchang/ubuntu-14.04-32bit'
```

Comment:
Looks like you could use the official `i386/ubuntu:14.04` image which is probaly kept more up-to-date than the erwinchang image?
https://hub.docker.com/_/ubuntu/
https://hub.docker.com/r/i386/ubuntu/